analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count-plot.Rmd

```diff
@@ -0,0 +1,507 @@
+---
+title: "Longitudinal analysis for the PBTA Cohort (pbta-tumor-evolution project)"
+author: 'Antonia Chroni <chronia@chop.edu> and Jo Lynne Rokita <rokita@chop.edu> for D3B'
+date: "2023"
+output:
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+---
+
+
+#### Tumor evolution project 
+
+# Background
+This is an exploratory analysis of longitudinal data in the PBTA cohort.
+We are investigating the number of histologies (cancer_type column): (1) per assay (experimental_strategy) and (2) pairs of genomic and transcriptomic assays.
+We are also looking at the number of patient samples available per histology with paired genomic and transcriptomic assays for (1) each time point (disease_stage) and (2) when Diagnosis is present and paired with another point in time.
+
+### Data used 
+In this notebook, we are looking into the v12 histologies file (last updates on May 1st, 2023) and subset to the PBTA cohort.
+We use tumor_descriptor and focus on time points recorded.
+
+### Note
+The colors in the plots do not match the ones of the PBTA cohort yet. ---TO DO---
+
+
+### Usage
+
+This notebook is intended to be run via the command line from the top directory
+of the repository as follows:
+
+```
+Rscript -e "rmarkdown::render('analyses/01_sample_distribution_analysis/01-tumor-descriptor-and-assay-count-plot.Rmd', clean = TRUE)"
+```
+
+# Set up
+
+```{r load-library}
+suppressPackageStartupMessages({
+  library(tidyverse)
+  library(ggplot2)
+  library(flextable)
+})
+```
+
+## Directories and File Inputs/Outputs
+
+```{r set-dir-and-file-names}
+# Detect the ".git" folder -- this will be in the project root directory.
+# Use this as the root directory to ensure proper sourcing of functions no
+# matter where this is called from
+
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+setwd(root_dir)
+<<<<<<< HEAD
+
+=======
+>>>>>>> add-sample-distribution
```

Comment:
looks like some code annotation got placed here on accident.

Reply:
thanks for catching this!